bin/prover-client/src/args.rs

```diff
@@ -173,4 +178,13 @@ impl Args {
 
         workers
     }
+
+    /// Resolves the rollup params file to use, from a path, and validates
+    /// it to ensure it passes sanity checks.
+    pub fn resolve_and_validate_rollup_params(&self) -> anyhow::Result<RollupParams> {
+        let json = fs::read_to_string(&self.rollup_params)?;
+        let rollup_params = from_str::<RollupParams>(&json)?;
+        rollup_params.check_well_formed()?;
+        Ok(rollup_params)
```

Comment:
Isn't file on disk is the whole `Params` structure?  We can extract the inner `RollupParams` from there.

Reply:
The file on disk only requires the RollupParams structure. Only RollupParams is set during the functional tests as well.